### PR TITLE
[v.2.0.0-api-changes] Applying api v2 changes

### PR DESCRIPTION
**Pull request by qzagarese**

### CODE REVIEW

File: src/main/java/com/github/dockerunit/annotation/WithSvc.java [hunk 1]
```diff
@@ -25,40 +25,40 @@
...
 
     /**
      * A prefix that will be used to give names to the containers instantiated
-     * to run your service.
+     * to run your svc.
      * <p>
      * If not set, Docker random names will be used.
      * If set and replicas has been set to 1, the actual value will be the container name.
-     * If set and replicas has been set to more than 1, containers will be named as value + "1", value + "2", ...
+     * If set and replicas has been set to more than 1, containers will be svcDefinition as value + "1", value + "2", ...
```
Comment (pnsantos) [inline src/main/java/com/github/dockerunit/annotation/WithSvc.java]: `svcDefinition` ?

File: src/main/java/com/github/dockerunit/annotation/WithSvc.java [hunk 2]
```diff
@@ -25,40 +25,40 @@
...
+public @interface WithSvc {
 
     /**
-     * The descriptor class that defines how to instantiate the service
+     * The descriptor class that defines how to instantiate the svc
      *
      * @return the descriptor type
      */
-    Class<?> service();
+    Class<?> svc();
```
Comment (pnsantos) [inline src/main/java/com/github/dockerunit/annotation/WithSvc.java:36]: Now that I looked at the PRs I wonder if this should be called `descriptor` or `svcDescriptor` :thinking: 

Anyway if it's too much of a chore to change it, leave it for 3.0 :wink:

File: src/main/java/com/github/dockerunit/internal/service/DefaultServiceBuilder.java [hunk 3]
```diff
@@ -46,8 +37,10 @@ public Service build(ServiceDescriptor descriptor, DockerClient client) {
         for (int i = 0; i < descriptor.getReplicas(); i++) {
             instances.add(createInstance(descriptor, client, i));
         }
-        return new Service(descriptor.getNamed()
-                .value(), instances, descriptor);
+        Svc svcDefinition = descriptor.getSvcDefinition();
+        String name = svcDefinition.value().length() > 0 ?
+                descriptor.getSvcDefinition().value() : descriptor.getSvcDefinition().name();
```
Comment (pnsantos) [inline src/main/java/com/github/dockerunit/internal/service/DefaultServiceBuilder.java]: https://github.com/dockerunit/dockerunit-consul/pull/6#discussion_r280140423

File: src/main/java/com/github/dockerunit/annotation/WithSvc.java (hunk 1 above)
Comment (qzagarese) [reply src/main/java/com/github/dockerunit/annotation/WithSvc.java]: Nice one! Bad refactoring.

File: src/main/java/com/github/dockerunit/annotation/WithSvc.java (hunk 2 above)
Comment (qzagarese) [reply src/main/java/com/github/dockerunit/annotation/WithSvc.java:36]: lol :-D

File: src/main/java/com/github/dockerunit/internal/service/DefaultServiceBuilder.java (hunk 3 above)
Comment (qzagarese) [reply src/main/java/com/github/dockerunit/internal/service/DefaultServiceBuilder.java]: Good one!

**Pull request merged**